cometd-java/cometd-java-benchmark/cometd-java-benchmark-client/src/main/java/org/cometd/benchmark/client/CometDLoadClient.java

```diff
@@ -781,7 +792,7 @@ public long convert(long measure) {
     }
 
     private void reset() {
-        histogram.reset();
+        allHistograms.forEach(AbstractHistogram::reset);
```

Comment:
Since we are using `Histogram` everywhere, replace `AbstractHistogram::reset` with just `Histogram::reset`.

Reply:
Ok.